docs/user_guides/benchmarking/docker.rst

```diff
@@ -0,0 +1,59 @@
+.. *benchmarking*\ docker:
+
+Running on Docker
+=================
+
+We support using Docker to run SDGym. This can help you avoid dependency
+issues when running SDGym on Windows as well as in environments where
+you don't have permission to install libraries.
+
+Basic Usage
+-----------
+
+To get started with SDGym on Docker, you should do the following:
+
+1. Pull the `latest SDGym
+   image <https://hub.docker.com/r/sdvproject/sdgym>`__ from DockerHub
+   by running:
+
+.. code:: bash
+
+    docker pull sdvproject/sdgym <tag>
```

Comment:
This should be `sdvproject/sdgym:<tag>` (`:` between the image name and the tag)